Readme.md

```diff
@@ -32,10 +32,33 @@ Many platforms require additional image processing beyond the EDK2 build.
 Any such steps should be documented (as a Readme.md), and any necessary helper
 scripts be contained, under said platform directory.
 
-Any contributions to this branch should be submitted via email to the
-edk2-devel mailing list with a subject prefix of `[platforms]`. See
-[Laszlo's excellent guide](https://github.com/tianocore/tianocore.github.io/wiki/Laszlo's-unkempt-git-guide-for-edk2-contributors-and-maintainers) for details
-on how to do this successfully.
+Any contributions to this branch should be submitted via GitHub Pull Request,
+or email to the edk2-devel mailing list with a subject prefix of `[platforms]`. See
+[Laszlo's excellent guide](https://github.com/tianocore/tianocore.github.io/wiki/Laszlo's-unkempt-git-guide-for-edk2-contributors-and-maintainers) for details on how to do this successfully.
+
+For details of who owns code in certain parts of the repo, see the CODEOWNERS and
+REVIEWERS files. Look in CONTRIBUTORS.md to find out people's names and their
+email addresses.
+
+In general, you should not privately email the maintainer. You should
+email the edk2-devel list, and Cc the package maintainers and
+reviewers.
+
+If the package maintainer wants to hand over the role to other people,
+the package maintainer should send the patch to update Maintainers.txt

```

Comment:
Maintainers.txt was just dropped

Reply:
Thanks. I've fixed it in the current revision.